bigtable-dataflow-parent/bigtable-beam-import/src/main/resources/log4j.properties

```diff
@@ -21,4 +21,6 @@ log4j.appender.stdout.Target=System.out
 log4j.appender.stdout.layout=org.apache.log4j.PatternLayout
 log4j.appender.stdout.layout.ConversionPattern=%d{yyyy-MM-dd HH:mm:ss} %-5p %c{1}:%L - %m%n
 
-log4j.category.org.apache.hadoop.io.compress.CodecPool=WARN
+log4j.category.org.apache.hadoop.io.compress.CodecPool=TRACE
```

Comment:
Is this intentional?  Can the log volume in TRACE mode flood the logs?

Reply:
No it is not, will revert this.